www/tutorial/index.html

```diff
@@ -0,0 +1,27 @@
+<html>
+  <head>
+    <title>InSpec Demo</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+    <!-- 1. Load libraries -->
+     <!-- Polyfill(s) for older browsers -->
+    <script src="node_modules/core-js/client/shim.min.js"></script>
+    <script src="node_modules/zone.js/dist/zone.js"></script>
+    <script src="node_modules/reflect-metadata/Reflect.js"></script>
+    <script src="node_modules/systemjs/dist/system.src.js"></script>
+    <!-- 2. Configure SystemJS -->
+    <script src="systemjs.config.js"></script>
```

Comment:
Is the systemjs.config.js not required. It is missing during my tests
<img width="1165" alt="screen shot 2016-08-31 at 10 18 14 am" src="https://cloud.githubusercontent.com/assets/1178413/18121483/08862f94-6f65-11e6-8c36-6f70c3f38ef9.png">